src/control/client/storage.go

```diff
@@ -58,6 +84,73 @@ func (c *control) listNvmeCtrlrs() (ctrlrs NvmeControllers, err error) {
 	return
 }
 
+// listScmModules prints all discovered Storage Class Memory modules installed.
```

Comment:
I am confused why we have such a function under a directory called client ...

Reply:
this is an unexported method, the gRPC message formats are distinct for the different type of storage devices (NVMe/SCM) therefore it makes sense to fetch from the different storage subsystems separately. the results are then aggregated in the client and presented in the return to ListStorage. we could do the aggregation on the gRPC server side if you would prefer ?

Reply:
Got it. I would then suggest to say:
"prints the aggregated list of all the discovered Storage Class Memory modules installed on the remote servers".